components/fieldgroups/experience-event/experienceevent-places-service-advanced.schema.json

```diff
@@ -0,0 +1,47 @@
+{
+  "meta:license": [
```

Comment:
If these two field groups are functionally same and the "advanced" version is just adding some properties on top of the other one, we should combine them into one and just enhance the description of the property.

Reply:
Also, we do have a data type already existing which contains the point of interest details.

https://github.com/adobe/xdm/blob/master/components/datatypes/placecontext.schema.json

This is used inside "Environment Details" Field Group. Please see if you can extend these field groups to add your additional attributes.